matchms/Scores.py

```diff
@@ -1,6 +1,12 @@
 from __future__ import annotations
+import json
+import pickle
```

Comment:
To my understanding, `setup.py` should include packages that have to be installed for a package to run properly. Since `json` and `pickle` are in Python's standard library, I don't think they belong in `setup.py`.